tests/test_csc.py

```diff
@@ -23,17 +23,22 @@
 import contextlib
 import pathlib
 import logging
-import time
 import unittest
 
+import pytest
+
 from lsst.ts import salobj
 from lsst.ts import mtmount
+from lsst.ts import utils
```

Comment:
A bit of nitpicking but wouldn't `from lsst.ts import salobj, mtmount, utils` make sense here?